src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1330,12 +1330,14 @@ private function registerAssetsConfiguration(array $config, ContainerBuilder $co
         }
     }
 
-    private function registerAssetMapperConfiguration(array $config, ContainerBuilder $container, PhpFileLoader $loader, bool $assetEnabled): void
+    private function registerAssetMapperConfiguration(array $config, ContainerBuilder $container, PhpFileLoader $loader, bool $assetEnabled, bool $httpClientEnabled): void
     {
         $loader->load('asset_mapper.php');
 
-        if (!$assetEnabled) {
-            $container->removeDefinition('asset_mapper.asset_package');
```

Comment:
I'm afraid this just broke something on 6.4. I'm having this error : `The service "asset_mapper.asset_package" has a dependency on a non-existent service "assets._default_package".`

Reply:
see #57589